server/controllers/customers/customer-controller.js

```diff
@@ -0,0 +1,44 @@
+const models = require('../../models')
+const { sendTwilioSMS } = require('../../utilities/send-sms')
+
+// Get all Cusotmers
+const getCustomers = (req, res) => {
+  models.Customer.findAll()
+    .then(data => res.send(data))
+    .catch(err => {
+      res.status(500).send(err.message || "Some error occured while retrieving customers")
+    })
+}
+
+// Send Message
+const sendMessage = async (req, res) => {
+  try {
+    const { id } = req.params
+    const { message } = req.body
+    const customer = await models.Customer.findOne({ where: { id } })
+
+    if (!customer || !message) {
+      res.status(404).send('Customer not found OR Message not provided')
+    }
+
+    // Save message to messages table
+    const newMessage = await models.Message.create({
+      text: message,
+      customerId: id,
+      dateSent: new Date(Date.now()).toISOString(),
+      sent: 1
```

Comment:
Why do we want to set it to 1 now? What if the twilio api request fails? We may want it to be one only if the sendTwilioSMS succeeds.